src/common/Utilities/Util.cpp

```diff
@@ -102,6 +102,45 @@ Tokenizer::Tokenizer(const std::string &src, const char sep, uint32 vectorReserv
     }
 }
 
+#if (defined(WIN32) || defined(_WIN32) || defined(__WIN32__))
+struct tm* localtime_r(time_t const* time, struct tm* result)
+{
+    localtime_s(result, time);
+    return result;
+}
+#endif
```

Comment:
actually, we already have this, but it's not ready merged to master
https://github.com/azerothcore/azerothcore-wotlk/pull/3365/files

Reply:
@Helias this part in fact must be in this PR and #3365 must be next after this